xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -541,17 +541,9 @@ private void handleStreamClosed(Status error) {
                 retryBackoffPolicy.nextBackoffNanos() - stopwatch.elapsed(TimeUnit.NANOSECONDS));
       }
       logger.log(Level.FINE, "{0} stream closed, retry in {1} ns", new Object[]{this, delayNanos});
```

Comment:
I think it will be useful to log the contents of `error` to help debug. Otherwise LGTM

Reply:
Improved. Added another log message at the beginning of handle stream closed method. It will print out whether closed by server calling `onComplete` or `onError` with cause.